markdown/setup/squirrel-driver-setup.md

```diff
@@ -0,0 +1,57 @@
+### SQuirreL SQL Client 
+[Link to product webpage](https://sourceforge.net/projects/squirrel-sql/).
+
+#### Adding the Amazon Timestream JDBC Driver
+
+1. Launch the SQuirreL SQL Client application.
+2. Ensure the **Drivers** tab is selected.
+
+    ![Example](../images/squirrel/squirrel1.png)
+
+3. Navigate to menu path **Drivers > New Driver ...**
+
+    ![Example](../images/squirrel/squirrel2.png)
+
+4. For the **Name:** field, enter **Timestream**.
+5. For the **Example JDBC URL:** field, enter `jdbc:timestream://Region=<region>;`
+6. Select the **JDBC Driver Class Path** tab.
+7. Click the **Add** button and navigate to the downloaded Amazon Timestream JDBC driver JAR file.
+8. Click **OK** to add the JAR file to the **JDBC Driver Class Path**.
+9. Click the **List Drivers** button.
+10. For the **Class Name:** field, ensure the `software.amazon.timestream.jdbc.TimestreamDriver` is selected.
+
+    ![Example](../images/squirrel/squirrel3.png)
+
+11. Click the **OK** button to create and save the driver settings.
+
+#### Connecting to Amazon Timestream Using SQuirreL SQL Client
+
+1. Launch the SQuirreL SQL Client application.
+2. Ensure the **Aliases** tab is selected.
+
+    ![Example](../images/squirrel/squirrel4.png)
+
+3. Navigate the menu path **Aliases > New Alias...**
+4. In the popped up window, fill the following info
+
+    a. For the **Name:** field, enter a name for this alias. For example, `TimestreamConnection`
+    
+    b. For the **Driver:** field, ensure **Timestream** is selected.
+
+    c. For the **URL:** field, enter your JDBC connection string. For example, `jdbc:timestream://Region=us-east-1;`
+
+    ![Example](../images/squirrel/squirrel5.png)
+
+    d. For the User **User Name:** field, enter your Amazon Timestream account ID.
+    
+    e. For the **Password:** field, enter the corresponding password for the acount ID.
```

Comment:
are we sure the username/password fields are needed? I think the driver currently does not have capabilities to read username/password and all credentials are either read from AWS configure or inside the connection string.

Reply:
Context: I leave the username/password fields for Aqua Data Studio empty and I'm still able to connect

Reply:
If user/password are not provided in the connection string, they could be provided here.

Reply:
Yea, but my main concern is that if user enters user/password there, the driver does not really handle it. Were you able to connect to driver using username/password and without the aws configuration set up?

Reply:
I tried the following step and tested that Aqua Data Studio password/username window does not work. Please confirm if it works with SQuirreL:
1. Open system environment variables, and edit `AWS_ACCESS_KEY_ID` to `AWS_ACCESS_KEY_ID-tmp`, and do the same for `AWS_SECRET_ACCESS_KEY` and rename it to `AWS_SECRET_ACCESS_KEY-tmp`. (e.g., `AWS_ACCESS_KEY_ID-tmp=<accessKeyId>`) This way the saved aws configuration will not be picked up by the JDBC driver.
2. Open BI tool and connect using username/password
3. Check if connection works. If connection does not work, then the driver cannot be connected using BI tool's username/password dialog and we'll need to tell user to ignore that dialog. This is ok because I don't think the old JDBC driver also supported this.

Reply:
Yes, you are right. The user/password does not work. They come from AWS configuration. Let me update this.